### PR TITLE
Csur 2024-12-11

### PR DESCRIPTION
**Pull request by hiletmis**
As part of https://github.com/api3dao/data-feeds/issues/1882 https://github.com/api3dao/tasks/issues/1547

### CODE REVIEW

File: chains/sonic.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "alias": "sonic",
+  "decimals": 18,
+  "explorer": {
+    "browserUrl": "https://explorer.soniclabs.com/"
```
Comment (bdrhn9) [inline chains/sonic.json:5]: Not working for now. We can check it later again.

Comment (hiletmis) [reply chains/sonic.json:5]: Seems like it works now along with testnet explorer

Comment (bdrhn9) [reply chains/sonic.json:5]: Still not working for me. Tried with different DNS, browser, VPN stuff.

**Review by bdrhn9 — APPROVED** [review]
We can start surveying without waiting for their block explorer to be fixed.

**Pull request merged**